README.md

```diff
@@ -63,46 +86,51 @@ The files/folder structure are a  **WIP** and may contain some unused files. The
 └── views/        // Individual Views
 ```
 
+### Working with language files
+
+Olympus uses linguijs to manage translation. https://github.com/lingui/js-lingui
+
+The language files are located in `src/locales`
+
+- Use `yarn lingui:extract` to scan the code for new translatable entries add them to the po files
+- Use `yarn lingui:compile` to compile the po files into the javascript files actually used by the application
```

Comment:
add a note about contacting the Discord Translations channel when you have a new feature that needs translation.

Reply:
also add a note telling devs to encapsulate strings that should be translated in `<Trans>`

Reply:
Done. Do not hesitate to tell me if my english is bad. I won't take it personaly.